### DISCUSSION TITLE
Create a couple of two halves and treat it as one window

### DISCUSSION THREAD

**Discussion by yochem** [rxhanson/Rectangle#429] [Ideas]
Okay, I don't know if this is possible but it would make life so much easier.

I often work with two windows, each covering the halve of the screen. An example usage would be a code editor and a latex document (pdf-viewer). But then I want look something up or change music in the spotify app or whatsoever. But after doing this, I need to first select the pdf viewer and then the code editor. Here's the screencast of it:

https://user-images.githubusercontent.com/23235841/113476859-6a96f400-947e-11eb-925f-3bc1a8aeb92b.mov

It would be much easier if the two windows (being iterm2 and brave in the example) could be coupled. Would this be possible in any way? What do you think?

**Comment by rxhanson** [comment]
My workflow in this type of situation is typically just to use cmd+h to hide an app like Spotify.

Although focusing windows programmatically is possible, there's a little more complexity than meets the eye for handling scenarios like having multiple windows of the same app open. 

I don't think this is something I would roll into Rectangle at this time. Worth considering if more window focusing features get rolled in at some point.

**Reply by yochem** [reply]
Thank you for your comment. I already thought that implementing this would be hard to do. I'll try to get cmd+h into my muscle memory ;) Cheers!
